src/validator.js

```diff
@@ -463,7 +460,7 @@ export default class Validator {
 
   _getAddonArchitecture(xpiMetadata) {
     // If we find a file named bootstrap.js this is assumed to be a
-    // Jetpack add-on: https://github.com/mozilla/amo-validator/blob/7a8011aba8bf8c665aef2b51eb26d0697b3e19c3/validator/testcases/jetpack.py#L154
+    // Jetpack add-on: https://github.com/mozilla/amo-validator/blob/7a8011a/validator/testcases/jetpack.py#L154
```

Comment:
Heh, I always worry eventually something will repeat the short hash, but I assume Git then picks the first hash in the history that uses it?

I should get over this fear! ^_^